eslint/babel-eslint-plugin/src/rules/object-curly-spacing.js

```diff
@@ -1,26 +1,20 @@
 import ruleComposer from "eslint-rule-composer";
 import eslint from "eslint";
 
-const objectCurlySpacingRule = new eslint.Linter()
-  .getRules()
-  .get("object-curly-spacing");
+const rule = new eslint.Linter().getRules().get("object-curly-spacing");
 
-export default ruleComposer.filterReports(
-  objectCurlySpacingRule,
-  (problem, metadata) => {
-    const node = problem.node;
+export default ruleComposer.filterReports(rule, problem => {
+  const node = problem.node;
 
-    // Allow `exportNamespaceFrom` and `exportDefaultFrom` syntax:
-    // export * as x from '...';
-    // export x from '...';
-    if (
-      node.type === "ExportNamedDeclaration" &&
-      node.specifiers.length > 0 &&
-      metadata.sourceCode.getTokenBefore(node.specifiers[0]).value === "export"
-    ) {
-      return false;
-    }
+  // Allow exportDefaultFrom syntax:
+  // export x from '...';
+  if (
+    node.type === "ExportNamedDeclaration" &&
+    node.specifiers.length === 1 &&
+    node.specifiers[0].type === "ExportDefaultSpecifier"
```

Comment:
In addition to the bug fix above, this code change is to allow the core rule to handle `exportNamespaceFrom`, since it's stage 4.